README.MD

```diff
@@ -25,6 +25,21 @@ inConfig(Compile)(Seq(
  ))
 ```
 
+##CSharp
```

Comment:
What about using this via precompiled NuGet package?

Reply:
Done